### DISCUSSION TITLE
What are the different download files for a release?

### DISCUSSION THREAD

**Discussion by octaeder** [texstudio-org/texstudio#2446] [Q&A] [answered]
Each release on the [releases page](https://github.com/texstudio-org/texstudio/releases/) offers different files like

1. texstudio-4.3.0beta2-osx.dmg
2. texstudio-4.3.0beta2-win-portable-qt5.zip
3. texstudio-4.3.0beta2-win-portable-qt6.zip
4. texstudio-4.3.0beta2-win-qt5.exe
5. texstudio-4.3.0beta2-win-qt6.exe
6. texstudio-4.3.0beta2-x86_64.AppImage 

I asssume all to be 64bit apps. 1) for Mac, 2) and 3) are portable versions for Win10 with different versions of Qt and 4) and 5) are the same, so installable versions. What is the last one?

Thx

**Comment by sunderme** [comment, marked as answer]
that is basically described on texstudio.org

**Reply by octaeder** [reply]
The AppImage is a fully packed image that should run on any Linux.-
